.github/workflows/build-cargo-sweep.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Cargo Sweep Binary
+
+on:
+  schedule:
+    # friday noon
+    - cron: '0 12 * * 5'
+    # or manually run
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Build
+        run: cargo install cargo-sweep
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Transfer Binary
+        run: 'mv ~/.cargo/bin/cargo-sweep ci/cargo-sweep'
+      - run: git pull
+      - name: Commit Binary
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "🤖 build cargo-sweep binary for ci use"
```

Comment:
Is this commit being pushed somewhere?

Reply:
yep, it pushes a built cargo-sweep binary, in case of a PR it's pushed to the PR's branch, in case of a push to master, it's pushed to the master branch

Reply:
Yes, this will push directly to the master branch

Reply:
Then it should fail right?
We have yewtempbot just to work around that fact.